test/middleware-config/express-entry.ts

```diff
@@ -33,22 +34,19 @@ async function startServer() {
     app.use(viteDevMiddleware)
   }
 
-  const middlewares = (await getMiddlewares()) as Middleware[]
-  middlewares.sort((m1, m2) => getOrder(m1.order) - getOrder(m2.order))
-  middlewares.forEach((middlewareSpec) => {
-    const middleware = middlewareSpec.value
-    app.all('*', createMiddleware(() => middleware)())
-  })
+  const middlewares = (await getMiddlewares()) as (RouteDefinition | MiddlewareDefinition)[]
+  const router = new UniversalRouter()
+  apply(router, middlewares)
+  app.all('*', createMiddleware(() => router[universalSymbol])())
+  // TODO replace with UniversalExpressRouter once done.
+  //      It should look like this:
+  // const router = new UniversalExpressRouter(app)
+  // apply(router, middlewares)
```

Comment:
I wonder if it could be further simplified to this:

```js
import { apply } from '@universal-middleware/express'
apply(app, middlewares)
```

Reply:
That would be the only `universal-middleware` code in this file (and inside `vike-express`).

Reply:
Indeed, that's even better

Reply:
https://github.com/vikejs/vike/pull/2050/files#diff-6a1963a7349ecacbba17183fc73dce759419880a402305d558aa187bc3ebeb03R35-R37 :sparkles: